src/utils/mergeIntervals.ts

```diff
@@ -0,0 +1,38 @@
+export function mergeIntervals(
+    intervals: { id: number; value: string; label: string }[],
+) {
+    if (intervals.length < 1) return [];
+    if (intervals.length < 2) return intervals;
```

Comment:
Well, i see that intervals are required in ts. So i guess we can merge this two conditions

```suggestion
    if (intervals.length < 2) return intervals;
```